map/views.py

```diff
@@ -1328,16 +1327,17 @@ def get_tile(self, req, ver, zoom, x, y,
                 draw = ImageDraw.ImageDraw(overlay)
                 box_corners = (0, 0, overlay_width, overlay_height)
                 ellipse_color = '#' + req.GET.get('lslgacolor', '#3388ff').lstrip('#')
-                draw.ellipse(box_corners, fill=None, outline=ellipse_color, width=3)
+                ellipse_width = int(np.round(float(req.GET.get('lslgawidth', 3)), 0))
+                draw.ellipse(box_corners, fill=None, outline=ellipse_color, width=ellipse_width)
 
                 rotated = overlay.rotate(PA, expand=True)
                 rotated_width, rotated_height = rotated.size
 
-                pix_from_left = (rahi - RA) * 3600 / pixscale
-                pix_from_top = (dechi - DEC) * 3600 / pixscale
+                ellipse_x = img_cx - ((RA - ra) * 3600 / pixscale * np.cos(np.deg2rad(dec)))
+                ellipse_y = img_cy - ((DEC - dec) * 3600 / pixscale)
 
```

Comment:
And here, you can go
```
ok,x,y = wcs.radec2pixelxy(RA, DEC)
```

These x,y coordinates will be 1 larger than what PIL expects (1,1) is the bottom-left pixel.

Reply:
Okay, thanks! One thing, what does `ok` indicate? (Would it make sense to only draw the ellipse `if ok`?)